modules/dedicated-cluster-install-deploy.adoc

```diff
@@ -124,12 +120,11 @@ spec:
         node-role.kubernetes.io/worker: ""
 ----
 
-.. Click *Create*. This creates the Cluster Logging Custom Resource and Elasticsearch Custom Resource, which you
-can edit to make changes to your cluster logging cluster.
+.. Click *Create*. This creates the Cluster Logging and Elasticsearch Custom Resources.
 
 . Verify the install:
```

Comment:
Maybe "Verify that the Pods for the Cluster Logging instance deployed:"

Reply:
Agreed! Added.